scripts/fetch_ci_binaries.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+"""Fetch executable binary artifacts from the latest Circle CI workflow on main"""
+
+import os
+import sys
+from typing import List, Optional, Tuple
+import requests
+
+HTTP_TIMEOUT = 5
+
+DEVNET_CI_URL = (
+    "https://circleci.com/api/v1.1/project/github/0xSpaceShard/starknet-devnet-rs"
+)
+
+BINARY_BUILD_JOB_PREFIX = "binary-build-"
+
+
+def get_artifact_info(job: dict) -> Optional[Tuple[str, str]]:
+    """
+    Returns the artifact (URL, name) corresponding to job object.
+    Returns None if job not a binary build.
+    """
+    if job["job_name"].startswith("binary-build-"):
+        job_id = job["job_id"]
+        job_name = job["job_name"]
+        artifact_name = (
+            job_name.replace(BINARY_BUILD_JOB_PREFIX, "starknet-devnet-") + ".tar.gz"
+        )
+        return (
+            f"https://output.circle-artifacts.com/output/job/{job_id}/{artifact_name}",
```

Comment:
Can we have all URLs at the beginning of the file? It will be easier to read and manage if url will change in the future.

Reply:
Refactored in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/423/commits/56f03619bded4008034a3e60859c8ddad3e88821